services/discovery/Makefile

```diff
@@ -104,12 +104,9 @@ $(BUILD_DIR): $(SRC_FILES) $(TEMPLATE_FILE)
 	sam build
 
 .PHONY: package
-package: $(PACKAGED_TEMPLATE_FILE)
-$(PACKAGED_TEMPLATE_FILE): $(BUILD_DIR)
-	@. ../../project.sh && cz_assert_profile && \
-	sam package \
-		--s3-bucket $(bucket) \
-		--output-template-file $@
+package: $(APP_ZIP)
```

Comment:
Here we manually zip up the lambda. We can still use `sam build` instead of our vendoring

Reply:
Is the packaged template file handled by `sam build`?

Reply:
We actually _don't_ want the packaged template b/c `sam package` replaces the `CodeUri` with its own.